homework/index.js

```diff
@@ -1,47 +1,142 @@
 'use strict';
 
-{
-  function fetchJSON(url, cb) {
+function fetchJSON(url) {
+  return new Promise((resolve, reject) => {
```

Comment:
Can you change this function to use `fetch` instead of `XMLHttpRequest`, as I asked in slack?